scripts/qesap/test/unit/test_qesap_terraform.py

```diff
@@ -200,3 +204,32 @@ def test_terraform_call_terraform_destroy(run, args_helper, config_yaml_sample):
     assert main(args) == 0
     run.assert_called()
     run.assert_has_calls(calls)
+
+
+@mock.patch("lib.process_manager.subprocess_run")
+def test_terraform_call_terraform_workspace(run, args_helper, config_yaml_sample):
```

Comment:
maybe renaming arg `run` as `subprocess_run` would make the code more clear for users not familiar with `mock.patch`, but since the same convention is used everywhere that's out of the scope of this PR